src/Qdrant.Client/Grpc/Condition.cs

```diff
@@ -0,0 +1,77 @@
+namespace Qdrant.Client.Grpc;
+
+/// <summary>
+/// A filter condition
+/// </summary>
+public partial class Condition
+{
+	/// <summary>
+	/// Evaluates whether the given <see cref="Condition"/> is <c>false</c>
+	/// </summary>
+	/// <param name="_">The condition.</param>
+	/// <returns>Always returns <c>false</c> so that both operands in boolean logical operators are evaluated.</returns>
+	public static bool operator false(Condition _) => false;
```

Comment:
I know true/false have to be overloaded in order to use `&&` on Conditions/Filters, but it's a bit odd for them to be defined (and arbitrarily always return false). After all, Condition/Filter aren't actually evaluatable in the C# context - they're just a representation meant to be sent elsewhere.

Note that these are necessary for `&&`, but not for `&`, so the alternative would be to remove these, and direct users to use `&`. That feels a cleaner to me and more aligned with the way C# operator overloading was meant to be done. And in any case, I suspect users will need to consult docs to know about the general operator overloading support (so the discoverability difference doesn't seem significant).

But if you feel strongly about it and would rather leave these in order to enable `&&`, I'm OK with that.

Reply:
I'll remove them for now, `&` and `|` do feel more aligned. 

I'm probably a little biased towards `&&` and `||` because they're overloaded in the Elasticsearch client so feel more familiar. We can always add them later if need be.